projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/cart-lib-checkout.migration.ts

```diff
@@ -0,0 +1,1719 @@
+    comment:
+      '// TODO:Spartacus - Method "getPaymentDetails" was removed from Class "CheckoutPaymentService". ',
+  },
+  {
+    class: 'CheckoutPaymentService',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'getSetPaymentDetailsResultProcess',
+    comment:
+      '// TODO:Spartacus - Method "getSetPaymentDetailsResultProcess" was removed from Class "CheckoutPaymentService". ',
+  },
+  {
+    class: 'CheckoutPaymentService',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'loadSupportedCardTypes',
+    comment:
+      '// TODO:Spartacus - Method "loadSupportedCardTypes" was removed from Class "CheckoutPaymentService". ',
+  },
+  {
+    class: 'CheckoutPaymentService',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'paymentProcessSuccess',
+    comment:
+      '// TODO:Spartacus - Method "paymentProcessSuccess" was removed from Class "CheckoutPaymentService". ',
+  },
+  {
+    class: 'CheckoutPaymentService',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'processStateStore',
+    comment:
+      '// TODO:Spartacus - Property "processStateStore" was removed from Class "CheckoutPaymentService". ',
+  },
+  {
+    class: 'CheckoutPaymentService',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'resetSetPaymentDetailsProcess',
+    comment:
+      '// TODO:Spartacus - Method "resetSetPaymentDetailsProcess" was removed from Class "CheckoutPaymentService". ',
+  },
+  {
+    class: 'CheckoutState',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'cardTypes',
+    comment:
+      '// TODO:Spartacus - PropertySignature "cardTypes" was removed from Interface "CheckoutState". ',
+  },
+  {
+    class: 'CheckoutState',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'orderType',
+    comment:
+      '// TODO:Spartacus - PropertySignature "orderType" was removed from Interface "CheckoutState". ',
+  },
+  {
+    class: 'CheckoutState',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'paymentTypes',
+    comment:
+      '// TODO:Spartacus - PropertySignature "paymentTypes" was removed from Interface "CheckoutState". ',
+  },
+  {
+    class: 'CheckoutState',
+    importPath: '@spartacus/checkout/core',
+    deprecatedNode: 'steps',
+    comment:
+      '// TODO:Spartacus - PropertySignature "steps" was removed from Interface "CheckoutState". ',
+  },
+  {
+    class: 'CheckoutOccEndpoints',
+    importPath: '@spartacus/checkout/occ',
+    deprecatedNode: 'loadCheckoutDetails',
+    comment:
+      '// TODO:Spartacus - PropertySignature "loadCheckoutDetails" was removed from Interface "CheckoutOccEndpoints". ',
+  },
+  {
+    class: 'CheckoutOccEndpoints',
+    importPath: '@spartacus/checkout/occ',
+    deprecatedNode: 'paymentTypes',
+    comment:
+      '// TODO:Spartacus - PropertySignature "paymentTypes" was removed from Interface "CheckoutOccEndpoints". ',
```

Comment:
not sure what to say about this since it moved to the b2b  occ checkout b2b endpoints, which is 'new'

Reply:
similarly to other properties related to b2b / scheduled-replenishment from this interface